### DISCUSSION TITLE
Upgrade win 11 24H2

### DISCUSSION THREAD

**Discussion by Mai-lem** [t1m0thyj/WinDynamicDesktop#624] [Q&A] [answered]
My computer updated to Windows 11 24H2 a few days ago, and since then the backgrounds do not change automatically and for them to change I have to restart my computer and I have even deleted the program and reinstalled it but the error continues.

**Comment by t1m0thyj** [comment]
This may be related to https://github.com/t1m0thyj/WinDynamicDesktop/issues/628 - will look into the 24H2 update incompatibilities when I have time, hopefully in the next week or two.

**Reply by t1m0thyj** [reply, marked as answer]
This issue seems to have been fixed by Windows updates - see https://github.com/t1m0thyj/WinDynamicDesktop/issues/601#issuecomment-2632499458
